keyboards/keychron/v4/iso/keymaps/mischback/rules.mk

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# SPDX-FileCopyrightText: 2023 Mischback <mischback.dev@googlemail.com>
+# SPDX-FileType: SOURCE
+
```

Comment:
license header not needed here.
```suggestion
```

Reply:
Fixed in 59b4e2672d491eb4058f8747240be47c61abb0da